src/main/scala/NIC.scala

```diff
@@ -2,17 +2,21 @@ package icenet
 
 import chisel3._
 import chisel3.util._
-import chisel3.experimental.{IO}
+import chisel3.experimental.{IO, DataMirror}
 import freechips.rocketchip.subsystem.{BaseSubsystem, TLBusWrapperLocation, PBUS, FBUS}
 import freechips.rocketchip.config.{Field, Parameters}
 import freechips.rocketchip.diplomacy._
 import freechips.rocketchip.regmapper.{HasRegMap, RegField}
 import freechips.rocketchip.tilelink._
 import freechips.rocketchip.util._
 import freechips.rocketchip.prci.{ClockSinkDomain}
-import testchipip.{StreamIO, StreamChannel, TLHelper, ClockedIO}
 import IceNetConsts._
 
+class ClockedIO[T <: Data](private val gen: T) extends Bundle {
```

Comment:
Can this be upstreamed to RC? I think this is useful / generic enough to be a util there.

Reply:
For this PR can you also add a comment that this is also duplicated in testchipip.